library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -375,8 +374,17 @@ private boolean updateLocked() {
         return isActive;
     }
 
+    private boolean isBatchStartingForTheFirstTime(DownloadBatch downloadBatch) {
```

Comment:
Still consistent as the check is done on top of the amount of bytes downloaded, equivalent to the old database check